package.json

```diff
@@ -48,7 +48,7 @@
     "@storybook/react": "^5.3.19",
     "@testing-library/react": "^10.4.7",
     "@types/enzyme": "^3.10.5",
-    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@types/enzyme-adapter-react-17": "npm:@types/enzyme-adapter-react-16@^1.0.6",
```

Comment:
specify to download directly from `npm`?

Reply:
https://www.npmjs.com/package/@types/enzyme-adapter-react-17
The package isn't out yet.